### DISCUSSION TITLE
真是服了antd5版本为啥要弃用菜单子组件Menu.Item，这样开发起来很麻烦，有没有大佬解释一下为啥

### DISCUSSION THREAD

**Discussion by sgwl163** [ant-design/ant-design#49812] [General]
真是服了antd5版本为啥要弃用菜单子组件Menu.Item，这样开发起来很麻烦，有没有大佬解释一下为啥

**Comment by ryanxc1107** [comment]
items 用法有何不妥？还是你就爱用 Menu.Item

**Reply by sgwl163** [reply]
> items 用法有何不妥？还是你就爱用 Menu.Item

不灵活，还需要重新遍历后才能到达想要的效果
